backend/src/config/config.ts

```diff
@@ -0,0 +1,12 @@
+export default () => ({
+  port: parseInt(process.env.PORT),
+  db: {
+    db_host: process.env.DATABASE_HOST,
+    db_port: parseInt(process.env.DATABASE_PORT),
```

Comment:
not sure if ports are required to be integers.
but tbh I don't really give no shit, so you can leave it like that lol